lib/pharos_cluster.rb

```diff
@@ -13,5 +13,5 @@ module Pharos
   KUBEADM_VERSION = ENV.fetch('KUBEADM_VERSION') { KUBE_VERSION }
   ETCD_VERSION = ENV.fetch('ETCD_VERSION') { '3.1.12' }
   DOCKER_VERSION = '1.13.1'
-  KUBELET_PROXY_VERSION = '0.3.5'
+  KUBELET_PROXY_VERSION = '0.3.6'
```

Comment:
Merge from #383?

Reply:
Yes, it did not make sense to keep these separate (that's why I closed #383)

Reply:
Ah, version bump is for the image push to `quay.io`?

Reply:
Yes.